docs/content/faq.md

```diff
@@ -144,5 +144,39 @@ The Azure mixins are going to be full fledged tools that communicate with the
 Azure APIs, and provide an improved user experience. It all depends on how much
 time you want to invest, and what you are starting from.
 
+## Does the Porter has its own Registry like DockerHub, ACR,Quay etc?
```

Comment:
Let's fix the capitalization of Registry, and remove the example registries so that the question is more straightforward.

```suggestion
## Does the Porter has its own registry?
```